doc/nrf/known_issues.rst

```diff
@@ -1346,7 +1346,37 @@ nRF5340 Audio
 
 The issues in this section are related to the :ref:`nrf53_audio_app` application.
 
-.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
+
+OCT-2070: Detection issues with USB-C to USB-C connection
+  Using USB-C to USB-C when connecting an nRF5340 Audio DK to PC is not correctly detected on some Windows systems.
+
+  **Affected platforms:** nRF5340 Audio DK
+
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+
+OCT-2154: USB audio interface does not work correctly on macOS
+  The audio stream is intermittent on the headset side after connecting the gateway to a Mac computer and starting audio stream.
+  This issue occurs sporadically after building the nRF5340 Audio application with the default USB port as the audio source.
+
+  **Affected platforms:** nRF5340 Audio DK
+
+.. rst-class:: v2-4-0 v2-3-0
+
+OCT-2325: If a problematic DFU-image is deployed, causing the image-swap at boot to fail,
+  the device may appear bricked with no obvious way of recovery.
```

Comment:
Does it only appear bricked, or is it actually bricked.
If only appearance: Workaround/fix?

Reply:
@pirun can you comment?

Reply:
In the current DFU infrastruction, once if we deployed NET firmware, there's no way to revert it.